commit 59e95c82ffee0de569efbf77e1206dc85ef34f9b
Author: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date:   Thu Apr 16 21:44:59 2020 +0200

    Really enable building rumpdisk
    
    * config.make.in (HAVE_LIBRUMP): Define.

diff --git a/config.make.in b/config.make.in
index efcbed4e..3b7af5e2 100644
--- a/config.make.in
+++ b/config.make.in
@@ -105,6 +105,9 @@ HAVE_LIBGCRYPT = @HAVE_LIBGCRYPT@
 # Whether we found liblwip.
 HAVE_LIBLWIP = @HAVE_LIBLWIP@
 
+# Whether we found librump.
+HAVE_LIBRUMP = @HAVE_LIBRUMP@
+
 # How to compile and link against liblwip.
 liblwip_CFLAGS = @liblwip_CFLAGS@
 liblwip_LIBS = @liblwip_LIBS@
